telemetry/definitions/commonDefinitions.json

```diff
@@ -3488,11 +3505,11 @@
             ]
         },
         {
-            "name": "codeTransform_mvnBuildFailed",
-            "description": "The maven command build failed.",
+            "name": "codeTransform_buildFailed",
```

Comment:
Generalizing this as you've done is a great idea. Any reason not to go a bit further and generalize it to `codeTransform_build` ? Then the "failed" state can simply be represented by the `result` field.

Using standard fields allows dashboards to better indicate the health of the clients. (Also it's a bit strange that `codeTransform_buildFailed` would have a `result=Success` value).

Reply:
Good point; will do that.